extensions/wikia/Insights/scripts/InsightsListTracking.js

```diff
@@ -27,8 +23,6 @@ define(
 			});
```

Comment:
Can we use on function and bind click event once? I will also use 'mousedown' event